web/themes/custom/recipe_theme/includes/block.inc

```diff
@@ -0,0 +1,8 @@
+<?php
+
+declare(strict_types = 1);
+
+/**
```

Comment:
The above issue means that the content from line 5-8 should go to the beginning of the file. :slightly_smiling_face: